cmr-exchange/metadata-proxy/src/cmr/metadata/proxy/concepts/granule.clj

```diff
@@ -43,40 +43,43 @@
         {:keys [bounding-box temporal page-num page-size]} params]
     (str "collection_concept_id=" coll-id
          (when (seq gran-ids)
-          (str "&"
-               (if exclude?
-                 (build-exclude gran-ids)
-                 (build-include gran-ids))))
+           (str "&"
+                (if exclude?
+                  (build-exclude gran-ids)
+                  (build-include gran-ids))))
          (when (seq bounding-box)
-          (str "&bounding_box="
-               (query-util/seq->str bounding-box)))
+           (str "&bounding_box="
+                (query-util/seq->str bounding-box)))
          (when (seq temporal)
-          (str "&"
-               (query-util/temporal-seq->cmr-query temporal)))
+           (str "&"
+                (query-util/temporal-seq->cmr-query temporal)))
          (when (seq page-num)
-          (str "&page-num=" page-num))
+           (str "&page-num=" page-num))
          (when (seq page-size)
-          (str "&page-size=" page-size)))))
+           (str "&page-size=" page-size)))))
 
 (defn async-get-metadata
   "Given a data structure with :collection-id, :granules, and :exclude-granules
   keys, get the metadata for the desired granules.
 
   Which granule metadata is returned depends upon the values of :granules and
   :exclude-granules"
-  [component search-endpoint user-token params]
+  [component search-endpoint user-token params sa-header]
   (let [url (str search-endpoint "/granules")
         payload (build-query component params)]
     (log/debug "Granules query CMR URL:" url)
     (log/debug "Granules query CMR payload:" payload)
     (request/async-post
      url
-     (-> {}
-         (request/add-token-header user-token)
-         (request/add-accept "application/json")
-         (request/add-form-ct)
-         (request/add-payload payload)
-         ((fn [x] (log/debug "Client request options:" x) x)))
+     (as-> {} n
```

Comment:
```suggestion
         (as-> {} request
       (request/add-token-header request user-token)
       (request/add-accept request "application/json")
       (request/add-form-ct request)
       (request/add-payload request payload)
       (if (some? sa-header)
         (request/add-search-after request sa-header)
         n)
       ((fn [x] (log/debug "Client request options:" x) x) request))
```

Reply:
Line 81 still has `n`, and indentation on 74 is off with the suggested change

Reply:
You are right, need to work on using that feature. Changing it to statement instead of code suggestion